samples/react/telehealth/src/views/DoctorView.tsx

```diff
@@ -79,8 +88,18 @@ function DoctorView(props: DoctorViewProps): JSX.Element {
             </header>
 
             <article className="flex flex-col grow overflow-hidden">
-              <MessageList />
-              <MessageInput sendButton={<ArrowUp />} />
+              <MessageList 
+                welcomeMessages = {{
+                message: { id: "id-welcome-d", type: "welcome", text: "Please open another window or tab to chat" },
+                  timetoken: (new Date().getTime() * 10000) + "",
```

Comment:
I suppose it's supposed to cast number as string? Why not simply `String(new Date().getTime() * 10000)` or I beliebe that `(new Date().getTime() * 10000).toString()` will also work

Reply:
Thanks @piotr-suwala , yes, I was just casting to a String.  I have made the suggested update.